### PR TITLE
Fix RN iOS example client app

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: react-native/client/example/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@coinbase/wallet-mobile-sdk": "file:..",
```
Comment (bangtoven) [inline react-native/client/example/package.json:13]: @vishnumad seems we need to keep this until we figure out how to import it more efficiently

Comment (vishnumad) [reply react-native/client/example/package.json:13]: This shouldn't be needed anymore. Importing of the sdk properly is already set up.

Comment (bangtoven) [reply react-native/client/example/package.json:13]: strange. on my machine, it couldn't find any of wallet-mobile-sdk package and its dependencies. cc @vishnumad

**Comment by bangtoven** [comment]
closing it for now. @vishnumad can you take a look why it didn't build ios

**Pull request closed without merging**